shopinvader_product/schemas/category.py

```diff
@@ -0,0 +1,58 @@
+# Copyright 2023 ACSONE SA/NV
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from __future__ import annotations
+
+from odoo.addons.extendable_fastapi import StrictExtendableBaseModel
+
+
+class ShortShopinvaderCategory(StrictExtendableBaseModel):
```

Comment:
@qgroulard since Shopinvader is already into the namespace, can you avoid to use the Shopinvader prefix for the schema classes and the method name. This would improve readability and comprehension.

ShopinvaderCategory -> ProductCategory
ShopinvaderVariant -> ProductProduct
ShopinvaderProduct -> ProductTemplate
.....